nuwa.json

```diff
@@ -0,0 +1,28 @@
+{
+    "response": [
+        {
+            "maintainer": "TheMysticle",
+            "oem": "Xiaomi",
+            "device": "Xiaomi 13 Pro",
+            "filename": "crDroidAndroid-15.0-20250511-nuwa-v11.4.zip",
+            "download": "https://sourceforge.net/projects/crdroid/files/nuwa/11.x/crDroidAndroid-15.0-20250511-nuwa-v11.4.zip/download",
+            "timestamp": 1746991152,
+            "md5": "40ff0398c1622bbf2ac8135c3c0da112",
+            "sha256": "24ef5405468eff68a727fb9f8a9d27d2e484994a9444f59ca30a859601156499",
+            "size": 2547848962,
+            "version": "11.4",
+            "buildtype": "",
```

Comment:
missing build type
Testing/Alpha/Beta/Weekly/Monthly
as we are currently not doing any BETA, should choose between Weekly/Monthly